draft-packages/stories/TextAreaField.stories.elm

```diff
@@ -0,0 +1,221 @@
+module Main exposing (TextFieldModel)
+
+import Browser.Dom exposing (blur)
+import CssModules exposing (css)
+import ElmStorybook exposing (storyOf, storybook)
+import Html exposing (Html, div)
+import Html.Attributes exposing (style)
+import KaizenDraft.Form.TextAreaField.TextAreaField as TextAreaField
+import Task
+
+
+type alias TextFieldModel =
+    String
+
+
+type TextFieldMsg
+    = TextFieldMsg String
+    | TextFieldEnter
+    | DoNothing
+
+
+model : TextFieldModel
+model =
+    ""
+
+
+update : TextFieldMsg -> TextFieldModel -> ( TextFieldModel, Cmd TextFieldMsg )
+update msg textFieldModel =
+    case msg of
+        TextFieldMsg value ->
+            ( value, Cmd.none )
+
+        TextFieldEnter ->
+            ( textFieldModel, Task.attempt (\_ -> DoNothing) (blur "the-id-field-input") )
+
+        DoNothing ->
+            ( textFieldModel, Cmd.none )
+
+
+storyContainer : List (Html msg) -> Bool -> Html msg
+storyContainer children reversed =
+    div [ style "width" "100vw", style "height" "100vh", styles.classList [ ( .reversedBackground, reversed ) ] ]
+        [ div [ style "padding" "24px" ]
+            children
+        ]
+
+
+main =
+    let
+        config =
+            { update = update
+            , init = ( model, Cmd.none )
+            , subscriptions = \_ -> Sub.none
+            }
+    in
+    storybook
+        [ storyOf "Default" config <|
+            \m ->
+                storyContainer
+                    [ TextAreaField.view
+                        (TextAreaField.default
+                            |> TextAreaField.id "the-id"
+                            |> TextAreaField.labelText "Your reply"
+                            |> TextAreaField.controlled False
+                            |> TextAreaField.autoComplete False
+                            |> TextAreaField.onBlurWithValue TextFieldMsg
+                            |> TextAreaField.onEnter TextFieldEnter
+                            |> TextAreaField.rows 4
+                            |> TextAreaField.description "Your reply will only be seen by you"
+                        )
+                    ]
+                    False
+        , storyOf "Default, Prefilled Value" config <|
+            \m ->
+                storyContainer
+                    [ TextAreaField.view
+                        (TextAreaField.default
+                            |> TextAreaField.id "the-id"
+                            |> TextAreaField.labelText "Your reply"
+                            |> TextAreaField.controlled True
+                            |> TextAreaField.autoComplete False
+                            |> TextAreaField.onBlurWithValue TextFieldMsg
+                            |> TextAreaField.textAreaValue "A prefilled value in uncontrolled mode"
```

Comment:
This is actually in controlled mode: `TextAreaField.controlled True`
```suggestion
                            |> TextAreaField.textAreaValue "A prefilled value in controlled mode"
```

There's not currently a way to prefill values in uncontrolled mode.
It's generally better to use uncontrolled unless you need controlled mode as typing can be more sluggish, and browser plugins like Grammarly will break (you need to add a `data-grammarly=false` or the field will be unusable for people using grammarly). This is fine for now, but I'd like us to make a follow up PR that allows you to prefill the element in uncontrolled mode (and we should favor the uncontrolled version in our app code)